cdisc_rules_engine/operations/base_operation.py

```diff
@@ -49,12 +49,12 @@ def __init__(
         self.evaluation_dataset = original_dataset
 
     @abstractmethod
-    def _execute_operation(self):
+    def _execute_operation(self, **kwargs):
```

Comment:
If we add kwargs to these methods, I think we'd have to add the argument to all operations. If that's the case, I think it is better to store the data we need from kwargs in the class instance. I.e set these values as instance variables when we create the instance of the operation instead of passing them to the operation execution method

Reply:
It seems that the rule instance variables are not passed to operation class. That is why I had to add it here.

Reply:
We should pass whatever instance variables we need to the operations class then